packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 ## v4.1.0-dev (Aug 8, 2024)
 
 ### New Features
-- Implement ability to set Shopper Context via search parameters in the Retail React App [#1986](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1986)
+- Implement ability to set Shopper Context via search paramters in the Retail React App [#1986](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1986)
+- Display an instagram promo banner from Page Designer in the PLP page of the Retail React App [#2016](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2016)
```

Comment:
This should work for all promotions right, we should leave out the word "instagram"

Reply:
Right that makes sense. All this change really does is add a page designer page to the PLP page